packages/pluggableWidgets/image-viewer-web/tests/e2e/specs/onClick.spec.ts

```diff
@@ -0,0 +1,56 @@
+import page from "../../../../../../configs/e2e/src/pages/page";
+import ImageViewer from "../objects/imageViewer.widget";
+import * as html from "../objects/constants";
+
+// eslint-disable-next-line jest/no-disabled-tests
+xdescribe("Image viewer", () => {
+    it("should trigger Microflow on click", () => {
+        page.open("p/onClickMicroflow");
+
+        const imageViewer = new ImageViewer("imageViewer1");
+        imageViewer.element.waitForDisplayed();
+        imageViewer.element.click();
+
+        browser.pause(1000);
+
+        page.modalDialog.waitForDisplayed();
+        expect(page.modalDialog.getText()).toContain("You clicked this image");
+    });
+
+    it("should trigger Nanoflow on click", () => {
```

Comment:
Just checking that one of the configured actions is already enough. We don't need to test the PW API. You can keep them now, but in the future you can save yourself some time ;)

Reply:
Yeah! I agree, this part I have imported from the old specs, still need to refactor based on the widget events from the other task which Chak is working on.